crates/zoon/src/element/button.rs

```diff
@@ -8,6 +8,60 @@ use std::marker::PhantomData;
 
 make_flags!(Label, OnPress);
 
+/// Html Button for your web page.
+/// You can create a new button by using its constructor and chain styling.
+///
+/// # Examples
+/// ```no_run
+/// use zoon::*;
+/// let button = Button::new().s(Align::center()).s(Padding::all(5));
+/// ```
+/// You can also create your button with specific events as well and update the styling the way you need by using [signals](https://crates.io/crates/futures-signals)
+///
+/// 1 - Create signal for local state management.
```

Comment:
- _"Create a signal"_ (I think there is `a` missing)
- Perhaps we could turn these points (1-3) into code comments in the code snippet below to move it closer to the related context.

Reply:
Yes , indeed the "a" is missing.

In my very first version, I had the comments inside the code, closer to context but the three steps were easier to read & memorized somehow as a "strategy" or "method" once grouped & structured together. I will rethink it and retry to see which one is best :smile:

Reply:
I updated :)